scripts/ts-api-extractor/src/adapters/out/ts-morph/parsers/type/declaration-source.ts

```
@@ -0,0 +1,73 @@
+/**
+ * Declaration source module
+ *
+ * Determines where symbols are declared (project, base-ui, react, dom, external).
+ */
+import type { Symbol } from 'ts-morph';
```

Biome (2.4.7)

[error] Do not shadow the global "Symbol" property.

(lint/suspicious/noShadowRestrictedNames)